src/Raven.Client/Documents/Session/SessionTimeSeriesBase.cs

```diff
@@ -124,6 +124,44 @@ public void Delete(DateTime? from = null, DateTime? to = null)
             }
         }
 
+        public void Increment(DateTime timestamp, IEnumerable<double> values)
+        {
+            if (Session.DocumentsById.TryGetValue(DocId, out DocumentInfo documentInfo) &&
+                Session.DeletedEntities.Contains(documentInfo.Entity))
+                ThrowDocumentAlreadyDeletedInSession(DocId, Name);
+
+            var op = new TimeSeriesOperation.IncrementOperation()
+            {
+                Timestamp = timestamp.EnsureUtc(),
+                Values = values.ToArray()
```

Comment:
We need to verify here and on the server side that the values are not negative

Reply:
Why, they *can* be negative.

Reply:
IIRC for negative we will need to have a `Dec` method and a special dbId tag... because upon conflict we will always take the higher value

Reply:
Damn, you are correct, didn't consider that.
But still, that is something that I would rather handle server side, not client side.
Better to have a `$"-{dbid}" instead.

Reply:
we decided to have only increment API, which will be used for decrement as well, so negative values are allowed.